src/index.html

```diff
@@ -5,8 +5,317 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header page__header">
+      <div class="logo header__logo">
+        <a href="#" class="logo__link">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a 
+              href="#" 
+              data-qa="nav-hover" 
+              class="nav__link"
+            >
+              Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Гаджеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Планшеты</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Фото</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <section class="section main__section">
+        <div class="card section__card" data-qa="card">
+          <div class="card__image" ></div>
+          <div class="card__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </div>
+        
+          <p class="card__code"> Код товара: 195434</p>
+        
+          <div class="card__rating">
+            <div class="stars stars--4">
+              <div class="stars__star "></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__feedback">Отзывов: 5</p>
+          </div>
+        
+          <div class="card__price">
+            <p class="card__price-text">Цена:</p>
+            <p class="card__price-value">69 999 грн</p>
+          </div>
+        
+          <a 
+            href="#" 
+            data-qa="card-hover" 
+            class="card__link card__link--hover"
```

Comment:
You should't add a modifier for `hover`, just use :hover in styles.
Change in all blocks.
```suggestion
            class="card__link"
```